packages/amplication-client/src/Resource/git/SyncWithGithubPage.tsx

```diff
@@ -82,13 +82,15 @@ const SyncWithGithubPage: React.FC = () => {
             resource={data.resource}
             onDone={handleOnDone}
             gitRepositorySelectedCb={gitRepositorySelectedCb}
+            gitRepositoryCreatedCb={handleOnDone}
```

Comment:
why do we need two props to hold the same value?

Reply:
`onDone={handleOnDone}` and `gitRepositoryCreatedCb={handleOnDone}`

Reply:
To trigger the refetch method in two different cases